editors/code/.vscode/settings.json

```diff
@@ -9,5 +9,6 @@
 		"dist": true // set this to false to include "dist" folder in search results
 	},
 	// Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-	"typescript.tsc.autoDetect": "off"
+	"typescript.tsc.autoDetect": "off",
+	"tabWidth": 4
```

Comment:
That's going to apply to all languages, like, even R files probably when you open a new one? Can we scope this to a language or something? I'm also not sure why we need this, the prettier config file should be working instead. You should not need the Zed config either unless something else is wrong.

Reply:
I think it was needed in json files or something like that. Using the same tab width across all languages in Air seems like a feature to me?